.github/workflows/release.yml

```diff
@@ -8,6 +8,24 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  lint-format:
+    name: Lint and format check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Ruff
+        run: pip install ruff
+      - name: Python format check (Ruff)
+        run: ruff format --check -v plugin/
+      - name: Python lint check (Ruff)
+        run: ruff check --output-format=github -v plugin/
```

Comment:
maybe also add `autopep8`?

Reply:
`ruff format` seems to fulfill the same use-case as autopep8. Do you have a preference for a certain one?

Reply:
iirc yt-dlp's hatch fmt --check does both and sometimes autopep8 has different behaviour from ruff